### PR TITLE
Added batch normalized critic v1 and changed default in TD3

### LINKED ISSUE #6: Add BatchNormalization to Critic
Batch normalization critic model

### PR DESCRIPTION
**Pull request by armenkasp**
Added batch normalization to critic to create critic-v1

Results from the unit test baselines and registry:

image

image

### CODE REVIEW

**Review by Kishanrajput — CHANGES_REQUESTED** [review]
I do not see a purpose of having a separate version of critic to include one extra layer. This could be made configurable in v0 itself with an entry in cfg file that act as conditional. Am I missing something?

**Comment by armenkasp** [comment]
> I do not see a purpose of having a separate version of critic to include one extra layer. This could be made configurable in v0 itself with an entry in cfg file that act as conditional. Am I missing something?

Removed the extra critic file and init call and moved the batch normalization into a conditional in the v0 critic with configuration control. Testing now.

**Comment by armenkasp** [comment]
Configuration flag now controls if batch normalization is used in the critic

Screenshot 2024-06-18 at 3 12 02 PM

**Review by Kishanrajput — APPROVED** [review]
